app/models/gsa18f/procurement.rb

```diff
@@ -24,6 +31,7 @@ def self.purchase_amount_column_name
       greater_than_or_equal_to: 1
     }, presence: true
     validates :product_name_and_description, presence: true
+    validates :purchase_type, presence: true
     validates :recurring_interval, presence: true, if: :recurring
 
     def self.relevant_fields(recurring)
```

Comment:
Do we need to add the `purchase_type` to this method so it gets displayed on the `show` page and in the emails?